tests/unit/test_dns.py

```diff
@@ -133,9 +144,13 @@ def test_dns_cache_negative(urlextract, dns_resolver):
     assert len(results) == 1
     assert len(dns_resolver.cache.data) == 2
 
-    results = urlextract.find_urls("https://invalid.invalid", check_dns=True)
```

Comment:
I need to check why invalid.invalid didnt work.  I remember this being a problem in development of dns-cache, and have tests about it there, but have forgotten the specifics.

Reply:
For the record, I believe this was because urlextract uses known TLD, and rejects invalid.invalid because it isnt in a TLD, so the DNS isnt even checked.

Thanks @lipoja for merging.